pages/code-of-conduct.tsx

```diff
@@ -10,89 +10,160 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
+      <li>Harassing photography or recording</li>
+      <li>Sustained disruption of talks or other events</li>
+      <li>Inappropriate physical contact</li>
+      <li>Unwelcome sexual attention</li>
+      <li>Advocating for, or encouraging, any of the above behaviour</li>
+    </ul>
+
+    <h2>Enforcement</h2>
+
+    <p>Participants asked to stop any harassing behavior are expected to comply immediately.</p>
+
     <p>
-      Email <a href={props.pageMetadata.conference.ContactEmail}>{props.pageMetadata.conference.ContactEmail}</a> or
-      call our designated contact person {props.pageMetadata.conference.EmergencyContactName} on{' '}
-      {props.pageMetadata.conference.EmergencyContactPhoneNumber}. You can also approach any one of the conference staff
-      who will be identifiable by the branded DDD Perth t-shirts they'll be wearing.
+      If a participant engages in harassing behaviour, event organisers retain the right to take any actions to keep the
+      event a welcoming environment for all participants. This includes warning the offender or expulsion from the
+      conference with no refund.
     </p>
 
-    <h2>The Quick Version</h2>
     <p>
-      Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of
-      gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race,
-      ethnicity, or religion (or lack thereof). We do not tolerate harassment of conference participants in any form.
-      Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties,
-      Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from
-      the conference <em>without a refund</em> at the discretion of the conference organisers.
+      Event organisers may take action to redress anything designed to, or with the clear impact of, disrupting the
+      event or making the environment hostile for any participants. We expect participants to follow these rules at all
+      event venues, event-related social activities as well as social media.
     </p>
 
-    <h2>The Less Quick Version</h2>
+    <h2>Reporting</h2>
+
     <p>
-      Harassment includes offensive verbal comments related to gender, gender identity and expression, age, sexual
-      orientation, disability, physical appearance, body size, race, ethnicity, religion, sexual images in public
-      spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of
-      talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+      If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Conference
+      staff can be identified by event branded t-shirts. Harassment and other Code of Conduct violations reduce the
```

Comment:
green, event-branded tshirts and organiser name tags?

Reply:
Works for me.

Reply:
I went with "event-branded tshirts and organiser name tags". Adding the t-shirt color made it specific to DDD Perth and as such would need to be parameterised to be of value to other conferences. I am fine with that but not sure of the value of adding the colour is. Conference staff are pretty prominent all event long.